widgets/HiveExport-action.json

```diff
@@ -2,6 +2,7 @@
   "metadata": {
     "spec-version": "1.0"
   },
+  "display-name": "Hive Bulk Export",
```

Comment:
Apache?

Reply:
although it might be too long for the plugin, drop the plugin on the canvas to verify

Reply:
Our other display names (Cassandra, for instance) don't have a leading "Apache".